From 2425a66a487076f48fe18b1e8f8b5d054a563148 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 11 Jul 2023 16:28:31 -0700
Subject: [PATCH 05/11] Enable stateless login

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 lib/getdef.c | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/lib/getdef.c b/lib/getdef.c
index dcd1fe7..413eeb4 100644
--- a/lib/getdef.c
+++ b/lib/getdef.c
@@ -489,6 +489,19 @@ static void def_load (void)
 	 */
 	def_loaded = true;
 
+	(void)putdef_str ("TTYGROUP", "tty");
+	(void)putdef_str ("HUSHLOGIN_FILE", ".hushlogin");
+	(void)putdef_str ("ERASECHAR", "0177");
+	(void)putdef_str ("KILLCHAR", "025");
+	(void)putdef_str ("SYS_UID_MIN", "500");
+	(void)putdef_str ("SYS_GID_MIN", "500");
+	(void)putdef_str ("ENCRYPT_METHOD", "SHA512");
+	(void)putdef_str ("DEFAULT_HOME", "yes");
+	(void)putdef_str ("USERGROUPS_ENAB", "yes");
+	(void)putdef_str ("CREATE_HOME", "yes");
+	(void)putdef_str ("MAIL_FILE", ".mail");
+	(void)putdef_str ("FORCE_SHADOW", "yes");
+
 #ifdef USE_ECONF
 
 	error = econf_readDirs (&defs_file, vendordir, sysconfdir, "login", "defs", " \t", "#");
-- 
2.41.0

