diff -Nurb /dev/null b/pam.d/chfn
--- /dev/null	1969-12-31 16:00:00.000000000 -0800
+++ b/pam.d/chfn	2018-03-27 13:29:06.324172430 -0700
@@ -0,0 +1,14 @@
+#
+# The PAM configuration file for the Shadow `chfn' service
+#
+
+# This allows root to change user infomation without being
+# prompted for a password
+auth		sufficient	pam_rootok.so
+
+# The standard Unix authentication modules, used with
+# NIS (man nsswitch) as well as normal /etc/passwd and
+# /etc/shadow entries.
+auth       include      common-auth
+account    include      common-account
+session    include      common-session
diff -Nurb /dev/null b/pam.d/chpasswd
--- /dev/null	1969-12-31 16:00:00.000000000 -0800
+++ b/pam.d/chpasswd	2018-03-27 13:29:06.324172430 -0700
@@ -0,0 +1,8 @@
+# The PAM configuration file for the Shadow 'chpasswd' service
+#
+
+auth       sufficient   pam_rootok.so
+auth       required     pam_unix.so
+account    required     pam_unix.so
+session    required     pam_unix.so
+password   include      common-password
diff -Nurb /dev/null b/pam.d/chsh
--- /dev/null	1969-12-31 16:00:00.000000000 -0800
+++ b/pam.d/chsh	2018-03-27 13:29:06.324172430 -0700
@@ -0,0 +1,19 @@
+#
+# The PAM configuration file for the Shadow `chsh' service
+#
+
+# This will not allow a user to change their shell unless
+# their current one is listed in /etc/shells. This keeps
+# accounts with special shells from changing them.
+auth       required   pam_shells.so
+
+# This allows root to change user shell without being
+# prompted for a password
+auth		sufficient	pam_rootok.so
+
+# The standard Unix authentication modules, used with
+# NIS (man nsswitch) as well as normal /etc/passwd and
+# /etc/shadow entries.
+auth       include      common-auth
+account    include      common-account
+session    include      common-session
diff -Nurb /dev/null b/pam.d/newusers
--- /dev/null	1969-12-31 16:00:00.000000000 -0800
+++ b/pam.d/newusers	2018-03-27 13:29:06.324172430 -0700
@@ -0,0 +1,4 @@
+# The PAM configuration file for the Shadow 'newusers' service
+#
+
+password   include      common-password
diff -Nurb /dev/null b/pam.d/passwd
--- /dev/null	1969-12-31 16:00:00.000000000 -0800
+++ b/pam.d/passwd	2018-03-27 13:29:06.324172430 -0700
@@ -0,0 +1,5 @@
+#
+# The PAM configuration file for the Shadow `passwd' service
+#
+
+password   include      common-password
